commons/src/main/java/org/infinispan/commons/marshall/UTF8Marshaller.java

```diff
@@ -0,0 +1,29 @@
+package org.infinispan.commons.marshall;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+
+import org.infinispan.commons.io.ByteBuffer;
+import org.infinispan.commons.io.ByteBufferImpl;
+
+public final class UTF8Marshaller extends AbstractMarshaller {
```

Comment:
I could make UTF8Marshaller (and the others) extend StringMarshaller, how does that sound?